eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -219,15 +160,41 @@ jobs:
         - pwsh: npm install -g moxygen
           displayName: Install Moxygen to generate markdown for docs.microsoft.com
 
+        - pwsh: azcopy copy "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/0.15.0/*" "$(System.DefaultWorkingDirectory)/parser" --recursive
+          displayName: Download ApiView Parser.
+
+        - task: Powershell@2
+          inputs:
+            filePath: $(System.DefaultWorkingDirectory)/eng/scripts/Generate-APIReview-Token-Files.ps1
+            arguments: >
+              -ArtifactList ('${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json | Select-Object Name)
+              -OutPath $(Build.ArtifactStagingDirectory)
+              -ParserPath $(System.DefaultWorkingDirectory)/parser/RelWithDebInfo/ParseAzureSdkCpp.exe
+              -ServicePath $(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}
+            pwsh: true
+            workingDirectory: $(Pipeline.Workspace)
+          displayName: Generate APIReview Token Files
+
         # Generate package spec JSON files for tagging the repo
         - ${{ each artifact in parameters.Artifacts }}:
             - pwsh: >-
                 New-Item
                 -ItemType directory
                 -Path $(Build.ArtifactStagingDirectory)
                 -Name packages/${{ artifact.Name }}
+                -Force
```

Comment:
Any specific reason to use force create directory?

Reply:
so it won't error out if the directory is already created, ran into that error when testing around with the pipeline